src/Microsoft.Identity.Client/Internal/MsalHelpers.cs

```diff
@@ -58,28 +58,14 @@ internal static string[] AsArray(this SortedSet<string> setOfStrings)
             return setOfStrings?.ToArray();
         }
 
-        internal static string AsSingleString(this SortedSet<string> setOfStrings)
+        internal static string AsSingleString(this IEnumerable<string> input)
```

Comment:
No point in having two overloads for this helper method - both _SortedSet_ and _string[]_ are enumerable.